lib/collection/script.js

```diff
@@ -62,6 +62,12 @@ _.assign(Script.prototype, /** @lends Script.prototype */ {
          * @type {string}
          */
         this.type = options.type || 'text/javascript';
+
+        /**
+         * @arguments {Script.prototype}
+         * @type {Array<Object>}
```

Comment:
This would provide more info about the interface:
```suggestion
         * @type {Array<{ id: string, name: string }>}
```